charter/index.html

```diff
@@ -291,92 +291,150 @@ <h1 id="title">
   </dl>
 </section>
 
-<section><h1>Community and Business Group Process</h1>
-  <p>
-    Terms of in this charter that conflict with those of the Community and
-    Business Group Process are void.
-  </p>
-  <section><h2>Work Limited to Charter Scope</h2>
-    <p>
-      The group will not publish Community Group Reports that are Specifications
-      on topics other than those listed under "Community Group Reports that are
-      Specifications" above. See below for how to modify the charter. The CLA
-      applies to these Community Group Reports.
-    </p>
+<section>
+    <h2>Community and Business Group Process</h2>
```

Comment:
I agree.  I took this from the charter template after making the changes in the template, so it was already reformatted there.